src/SDKs/EdgeGateway/Microsoft.Azure.Management.EdgeGateway/Microsoft.Azure.Management.EdgeGateway.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.EdgeGateway</PackageId>
+    <Description>Microsoft Azure Management EdgeGateway library</Description>
+    <AssemblyName>Microsoft.Azure.Management.EdgeGateway</AssemblyName>
+    <Version>1.0.0</Version>
```

Comment:
We recommend starting versioning from `0.9.0-preview` before releasing a stable version. Please confirm whether you are ready to release a stable version of the SDK?

Reply:
Changed to 0.9.0-preview. May be we will release first version after 1-2 weeks